Testscripts/Windows/VERIFY-XDP-LOAD-UNLOAD.ps1

```diff
@@ -0,0 +1,183 @@
+        Provision-VMsForLisa -allVMData $allVMData -installPackagesOnRoleNames "none"
+
+        # Generate constants.sh and write all VM info into it
+        Write-LogInfo "Generating constants.sh ..."
+        $constantsFile = "$LogDir\constants.sh"
+        Set-Content -Value "# Generated by Azure Automation." -Path $constantsFile
+        Add-Content -Value "ip=$($clientVMData.InternalIP)" -Path $constantsFile
+        Add-Content -Value "nicName=eth1" -Path $constantsFile
+        foreach ($param in $currentTestData.TestParameters.param) {
+            Add-Content -Value "$param" -Path $constantsFile
+        }
+        Write-LogInfo "constants.sh created successfully..."
+        Write-LogInfo (Get-Content -Path $constantsFile)
+        $installXDPCommand = @"
+./XDPDumpSetup.sh 2>&1 > ~/xdpConsoleLogs.txt
+. utils.sh
+collect_VM_properties
+"@
+        Set-Content "$LogDir\StartXDPSetup.sh" $installXDPCommand
+        Copy-RemoteFiles -uploadTo $clientVMData.PublicIP -port $clientVMData.SSHPort `
+            -files "$constantsFile,$LogDir\StartXDPSetup.sh" `
+            -username $user -password $password -upload -runAsSudo
+
+        Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+            -username $user -password $password -command "chmod +x *.sh" -runAsSudo | Out-Null
+        $testJob = Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+            -username $user -password $password -command "./StartXDPSetup.sh" `
+            -RunInBackground -runAsSudo
+        # Terminate process if ran more than 5 mins
+        # TODO: Check max installation time for other distros when added
+        $timer = 0
+        while ((Get-Job -Id $testJob).State -eq "Running") {
+            $currentStatus = Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+                -username $user -password $password -command "tail -2 ~/xdpConsoleLogs.txt | head -1" -runAsSudo
+            Write-LogInfo "Current Test Status: $currentStatus"
+            Wait-Time -seconds 20
+            $timer += 1
+            if ($timer -gt 15) {
+                Throw "XDPSetup did not stop after 5 mins. Please check logs."
+            }
+        }
+
+        $currentState = Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+            -username $user -password $password -command "cat state.txt" -runAsSudo
+        if ($currentState -imatch "TestCompleted") {
+            Write-LogInfo "XDPSetup successfully ran on $($clientVMDAta.RoleName)"
+
+            # Run XDP Load unload 3 times
+            $count_unload = 3
+            $curr_count = 0
+            while ( $curr_count -lt $count_unload) {
+                $testResult = Ping-XDPDump $clientVMData $serverVMData.SecondInternalIP "eth1" "_$curr_count"
+                $curr_count++
+            }
+
+        }   elseif ($currentState -imatch "TestAborted") {

```

Comment:
151 and 154 have extra spaces

Reply:
I guess line 157 and 160 have less spaces. Anyways I fixed that.